.github/workflows/fornightly-release.yml

```diff
@@ -0,0 +1,146 @@
+          echo "ARCH=x86_64"  >> $GITHUB_ENV
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Create tag
+        uses: actions/github-script@v5
+        with:
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/tags/${{ env.TAG_NAME }}',
+              sha: context.sha
+            })
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        with:
+          tag_name: ${{ env.TAG_NAME }}
+          release_name: ${{ env.RELEASE_NAME }}
+          draft: false
+          prerelease: false
+
+      - name: Build RPM packages
+        id: rpm_build
+        env:
+          _VERSION_ : ${{ env.RELEASE_NAME }}
+          _RELEASE_: ${{ env.RELEASE_NAME }}
+          _COMMITTER_: ${{ env.COMMITTER }}
+          _CHANGELOG_: ${{ env.CHANGE_LOG }}
+          _ARCH_: ${{ env.ARCH }}
+        run: |
+          make containerized_build_rpm
+          make containerized_build_container_rpm
+          cd _output/rpmbuild/
+          sudo tar czvf /tmp/kepler.rpm.tar.gz RPMS/
+          
+      - name: release rpm
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: /tmp/kepler.rpm.tar.gz
+          asset_name: kepler.rpm.tar.gz
+          asset_content_type: application/gzip
+      
+  
+  push-image:
+    name: Push image to the registry
+    needs: [build]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Login to Quay
+      uses: docker/login-action@v1
+      with:
+        registry: quay.io/sustainable_computing_io
+        username: ${{ secrets.BOT_NAME }}
+        password: ${{ secrets.BOT_TOKEN }}
+
+    - name: Build and push
+      uses: docker/build-push-action@v4
+      with:
+          context: .
+          platforms: linux/amd64
+          push: true
+          tags: quay.io/sustainable_computing_io/kepler:latest, quay.io/sustainable_computing_io/kepler:${{ env.RELEASE_NAME }}
+          labels: latest,  ${{ env.RELEASE_NAME }}
+          file: build/Dockerfile
+
+    - name: Generate SBOM
+      uses: anchore/sbom-action@v0.14.2
+      with:
+        image: quay.io/sustainable_computing_io/kepler:${{ env.RELEASE_NAME }}
+        artifact-name: sbom-kepler- ${{ env.RELEASE_NAME }}.json
+        output-file: ./sbom-kepler- ${{ env.RELEASE_NAME }}.spdx.json
+
+    - name: Attach SBOM to release
+      uses: softprops/action-gh-release@v1
+      with:
+        tag_name: ${{ env.TAG_NAME }}
+        files: ./sbom-kepler- ${{ env.RELEASE_NAME }}.spdx.json
```

Comment:
where are we publish those SBOM files?

Reply:
As of now we just put it as a part of release artifacts.

https://github.com/sustainable-computing-io/kepler/releases/tag/v0.5.1